interface/forms/gad7/view.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+/**
+ * gad-7 form using form api     view.php
+ * open a previously completed GAD-7 form for further editing
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Ruth Moulton <moulton ruth@muswell.me.uk>
+ * @copyright Copyright (c) 2021 ruth moulton <ruth@muswell.me.uk>
+ *
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+include_once("gad7.inc.php");  // common strings etc
```

Comment:
- add to this comment that this require_once calls does "call to globals.php" so we don't get confused in the future
- also use require_once
- also place a newline here to have a line separating above block from below block